src/App.vue

```diff
@@ -15,9 +15,9 @@
 		</NcAppNavigation>
 		<NcAppContent>
 			<div v-if="currentAppID" style="height: 100%">
-				<iframe height="100%" width="100%" :src="generateUrl(`/apps/ocs_api_viewer/view/${currentAppID}`)" />
```

Comment:
The controller method of this route still exists, please remove it (unless we need to keep the iframe to fix the styling.